site/templates/footer/footer.jet

```diff
@@ -0,0 +1,14 @@
+{{ import "./brand.jet" }}
+{{ import "./info.jet" }}
+{{ import "./nav.jet" }}
+
+{{ block footer() }}
+  {*{ ID is used for skip links. }*}
+  <footer id="footer">
+    {{ yield footerBrand() }}
+
+    {{ yield footerNav(navLinks=site.Navigation.Footer) }}
+
+    {{ yield footerInfo() }}
```

Comment:
Using [Include](https://github.com/CloudyKit/jet/wiki/3.-Jet-template-syntax#include-statement) instead of import+yield might save you from having to come up with block names.

Reply:
I'd rather use only blocks for consistency as some things need to be blocks... it looks tidier when everything is a block.